pyswmm/__init__.py

```diff
@@ -17,7 +17,8 @@
 from pyswmm.system import SystemStats
 from pyswmm.raingages import RainGages, RainGage
 
-VERSION_INFO = (0, 6, 3, 'dev0')
+VERSION_INFO = (1, 0, 0, 'dev0')
```

Comment:
@jennwuu can you confirm the versions here?

Reply:
@bemcdonnell the SWMM version?

Reply:
@bemcdonnell I think these version information should removed since swmm is from swmm-toolkit package

Reply:
@bemcdonnell *poke*